addons/web_editor/static/src/js/editor/add_snippet_dialog.js

```diff
@@ -260,6 +267,9 @@ export class AddSnippetDialog extends Component {
                 });
             }));
 
+            const snippetName = snippetPreviewWrapEl.querySelector("[data-name]")?.dataset?.name || _t("Select this block");
+            snippetPreviewWrapEl.setAttribute("title", snippetName);
```

Comment:
In fact, I'm not sure about this title. See:

![image](https://github.com/user-attachments/assets/d609e991-a024-4cc8-af59-7fe4833d1e0b)

I think it's not really ideal to have this text block covering a part of the preview. IMO it should be displayed outside using our popover ...
This one:
![image](https://github.com/user-attachments/assets/ac2c5a08-5a7f-4ba6-935e-81d6503eaf8e)
What do you think ?

Reply:
Your `title` overlay it's quite prominent indeed 🤔
On my side it's much more subtle, it may be relate to the OS .

|  |
|--------|
| ![image](https://github.com/user-attachments/assets/9c63525d-852e-41df-a73b-2d7c14ff4093) | 

Adding a "real tooltip" wasn't planned (@lebl-odoo from the future, wdyt?). 
I was thinking to show it in`/?debug` mode only, but importing debug_utils  just for that would be overkill 😕

I'll drop the feature for now... thanks for the feedback 👍

Reply:
Hi @stefanorigano , you meant using the "dark" tooltip inside the snippets modal? Is there technical limitations you're aware of, @bvr-odoo ? If no, I can create a task to ensure coherence between all screens of the website builder.

Reply:
Hello @lebl-odoo 👋 
I don't think that there are tech. limitations, it's more about usability and UX. We found the subtle `title` useful while working on the creation of snippets, but it's true that it could be useless/distracting for the general audience.

At the same time, this name it's visible also in the sidebar => maybe some users may find the correlation useful?
![image](https://github.com/user-attachments/assets/0862bac1-a13d-49c0-aab4-a4fc42394939)

As said we're not sure and it's probably not a big deal, just wanted to add you in the loop in case we were missing something 😉

Reply:
Perfect @stefanorigano. Actually, I found useful to display the name of the snippets somewhere in the modal otherwise users will struggle to know which keywords to use in the search bar.

Reply:
@lebl-odoo I’m also thinking about snippets like the 'Parallax' where there is no information about what it is (it’s just an image displayed in the modal) until you drag and drop it onto the page. Even though the word 'Parallax' probably won't provide more information to some people :wink:

Reply:
Indeed, we could imagine adding an additional description in the tooltip.
Let's update the [follow-up task](https://www.odoo.com/odoo/project/974/tasks/4072655) if it's suitable for you, @bvr-odoo

Reply:
I had a chat with FP about this, he wan't tooltip unfortunately